diff --git a/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.html b/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.html
index 2cd0dd657..0302f2d4c 100644
--- a/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.html
+++ b/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.html
@@ -1,4 +1,13 @@
 @if (pagination.totalItems === 0) {
+  <my-alert type="info">
+    @if (pluginType == 1) {
+      Plugins
+    } @else {
+      Themes
+    }
+    are being managed with Nix.
+  </my-alert>
+
   <div class="no-results">
     {{ getNoResultMessage() }}
   </div>
diff --git a/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.ts b/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.ts
index a4ad8021d..7233d2321 100644
--- a/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.ts
+++ b/client/src/app/+admin/plugins/plugin-list-installed/plugin-list-installed.component.ts
@@ -10,6 +10,7 @@ import { ButtonComponent } from '../../../shared/shared-main/buttons/button.comp
 import { DeleteButtonComponent } from '../../../shared/shared-main/buttons/delete-button.component'
 import { InfiniteScrollerDirective } from '../../../shared/shared-main/common/infinite-scroller.directive'
 import { PluginCardComponent } from '../shared/plugin-card.component'
+import { AlertComponent } from '@app/shared/shared-main/common/alert.component'
 
 @Component({
   selector: 'my-plugin-list-installed',
@@ -19,7 +20,8 @@ import { PluginCardComponent } from '../shared/plugin-card.component'
     InfiniteScrollerDirective,
     PluginCardComponent,
     ButtonComponent,
-    DeleteButtonComponent
+    DeleteButtonComponent,
+    AlertComponent
   ]
 })
 export class PluginListInstalledComponent implements OnInit {
