diff --git a/Cargo.lock b/Cargo.lock
index 26cb8e4bd..317145f77 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1676,7 +1676,7 @@ checksum = "f7012b1bbb0719e1097c47611d3898568c546d597c2e74d66f6087edd5233ff4"
 [[package]]
 name = "libmcaptcha"
 version = "0.2.3"
-source = "git+https://github.com/mCaptcha/libmcaptcha?tag=0.2.3#6bc0d11f8839d18c4d930cac0c0b86505ed1b2f6"
+source = "git+https://github.com/ngi-nix/libmcaptcha?branch=fix-pow_sha256-version#33eeff07f5281a693c75bd07d9bb75534ef48da3"
 dependencies = [
  "actix",
  "crossbeam-channel",
@@ -1684,7 +1684,7 @@ dependencies = [
  "derive_more",
  "log",
  "num_cpus",
- "pow_sha256 0.3.1 (git+https://github.com/mcaptcha/pow_sha256)",
+ "pow_sha256",
  "pretty_env_logger",
  "rand",
  "redis",
@@ -1809,7 +1809,7 @@ dependencies = [
  "mime_guess",
  "num_cpus",
  "openssl",
- "pow_sha256 0.3.1 (git+https://github.com/mcaptcha/pow_sha256?tag=0.3.1)",
+ "pow_sha256",
  "pretty_env_logger",
  "rand",
  "rust-embed",
@@ -2299,18 +2299,6 @@ dependencies = [
  "sha2",
 ]
 
-[[package]]
-name = "pow_sha256"
-version = "0.3.1"
-source = "git+https://github.com/mcaptcha/pow_sha256#148f1cb70d19114d1340661a77b2b679e95715f6"
-dependencies = [
- "bincode",
- "derive_builder",
- "num",
- "serde 1.0.147",
- "sha2",
-]
-
 [[package]]
 name = "ppv-lite86"
 version = "0.2.17"
diff --git a/Cargo.toml b/Cargo.toml
index 25058093c..3e7268f6c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -59,7 +59,7 @@ log = "0.4"
 lazy_static = "1.4"
 
 
-libmcaptcha = { version = "0.2.3", git = "https://github.com/mCaptcha/libmcaptcha", features = ["full"], tag ="0.2.3" }
+libmcaptcha = { version = "0.2.3", git = "https://github.com/ngi-nix/libmcaptcha", features = ["full"], branch ="fix-pow_sha256-version" }
 #libmcaptcha = { branch = "master", git = "https://github.com/mCaptcha/libmcaptcha", features = ["full"] }
 #libmcaptcha = { path = "../libmcaptcha", features = ["full"]}
 
@@ -72,8 +72,8 @@ mime = "0.3.16"
 num_cpus = "1.13.1"
 
 lettre = { version = "0.10.0-rc.3", features = [
-"builder", 
-"tokio1", 
+"builder",
+"tokio1",
 "tokio1-native-tls",
 "smtp-transport"
 ]}
diff --git a/db/db-core/Cargo.toml b/db/db-core/Cargo.toml
index 9b62a7eac..50253c841 100644
--- a/db/db-core/Cargo.toml
+++ b/db/db-core/Cargo.toml
@@ -8,12 +8,12 @@ documentation = "https://mcaptcha.org/docs/"
 license = "AGPLv3 or later version"
 authors = ["realaravinth <realaravinth@batsense.net>"]
 
-[dependencies] 
+[dependencies]
 async-trait = "0.1.51"
 thiserror = "1.0.30"
 serde = { version = "1", features = ["derive"]}
 url = { version  = "2.2.2", features = ["serde"] }
-libmcaptcha = {  version = "0.2.3", git = "https://github.com/mCaptcha/libmcaptcha", features = ["minimal"], default-features = false, tag = "0.2.3"}
+libmcaptcha = { version = "0.2.3", git = "https://github.com/ngi-nix/libmcaptcha", features = ["minimal"], branch ="fix-pow_sha256-version" }
 #libmcaptcha = { branch = "master", git = "https://github.com/mCaptcha/libmcaptcha", features = ["full"] }
 
 [features]
