diff --git a/src/settings.rs b/src/settings.rs
index e64b8a67..a2ab0e0f 100644
--- a/src/settings.rs
+++ b/src/settings.rs
@@ -164,6 +164,14 @@ impl Settings {

         s.merge(Environment::with_prefix("MCAPTCHA").separator("_"))?;

+        // Incomplete workaround for https://github.com/mCaptcha/mCaptcha/issues/105. There are
+        // other settings that still don't work, but this is the only one we need in order to securely
+        // package mCaptcha on NixOS.
+        if let Ok(val) = env::var("MCAPTCHA_SERVER_COOKIE_SECRET") {
+            s.set("server.cookie_secret", val).unwrap();
+            log::info!("Overriding [server].cookie_secret with environment variable");
+        }
+
         check_url(&s);

         if let Ok(val) = env::var("PORT") {
