diff --git a/weblate/utils/checks.py b/weblate/utils/checks.py
index 3b97c8849c..1b9b987093 100644
--- a/weblate/utils/checks.py
+++ b/weblate/utils/checks.py
@@ -231,7 +231,9 @@ def check_celery(app_configs, **kwargs):
                     weblate_check(
                         "weblate.E034",
                         "The Celery process is outdated or misconfigured."
-                        " Following items differ: {}".format(", ".join(differing)),
+                        + " Following items differ: {}".format(", ".join(differing))
+                        + str(current["vcs"])
+                        + str(pong["vcs"]),
                     )
                 )
         except TimeoutError:
