diff --git a/setup.py b/setup.py
index 6c851e5..b7b834c 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,6 @@
 import os
 from distutils.command.build import build
 
-from django.core import management
 from setuptools import find_packages, setup
 
 try:
@@ -15,8 +14,13 @@ except FileNotFoundError:
 
 class CustomBuild(build):
     def run(self):
-        management.call_command("compilemessages", verbosity=1)
-        build.run(self)
+        try:
+            from django.core import management
+
+            management.call_command("compilemessages", verbosity=1)
+            build.run(self)
+        except ModuleNotFoundError:
+            return
 
 
 cmdclass = {"build": CustomBuild}
@@ -31,3 +35,4 @@ setup(
     author="Tobias Kunze",
     author_email="r@rixx.de",
+    python_requires=">=3.7",
     license="Apache Software License",
