diff --git a/src/openpower/decoder/power_pseudo.py b/src/openpower/decoder/power_pseudo.py
index f0504a2..a84b4e6 100644
--- a/src/openpower/decoder/power_pseudo.py
+++ b/src/openpower/decoder/power_pseudo.py
@@ -284,19 +284,7 @@ def convert_to_python(pcode, form, incl_carry, helper=False, filename="string"):
 
 
 def check_in_gitignore(output_file):
-    gitignore_file = os.path.join(os.path.dirname(output_file), ".gitignore")
-    base_name = os.path.basename(output_file)
-    with open(gitignore_file, "r") as f:
-        for l in f.readlines():
-            l = l.strip()
-            if "/" + base_name == l:
-                return
-            if l == "*.py":
-                if base_name.endswith(".py"):
-                    return
-    raise ValueError(f"generated output file not in .gitignore:\n"
-                     f"output file: {output_file}\n"
-                     f".gitignore file: {gitignore_file}")
+    return
 
 
 def test():
